crates/test-utils/src/bin/dump-katana.rs

```diff
@@ -1,28 +1,15 @@
 use std::collections::HashMap;
 
-use ethers::abi::Token;
 use git2::{Repository, SubmoduleIgnore};
 use kakarot_rpc_core::client::api::KakarotStarknetApi;
 use kakarot_test_utils::constants::STARKNET_DEPLOYER_ACCOUNT_PRIVATE_KEY;
-use kakarot_test_utils::deploy_helpers::{
-    ContractDeploymentArgs, DeployerAccount, KakarotTestEnvironmentContext, TestContext,
-};
+use kakarot_test_utils::deploy_helpers::{DeployerAccount, KakarotTestEnvironmentContext};
 use starknet::accounts::Account;
 
 #[tokio::main]
 async fn main() {
     // Deploy all kakarot contracts + EVM contracts
-    let mut test_context = KakarotTestEnvironmentContext::new(TestContext::PlainOpcodes).await;
-    test_context = test_context
-        .deploy_evm_contract(ContractDeploymentArgs {
-            name: "ERC20".into(),
-            constructor_args: (
-                Token::String("Test".into()),               // name
-                Token::String("TT".into()),                 // symbol
-                Token::Uint(ethers::types::U256::from(18)), // decimals
-            ),
-        })
-        .await;
+    let test_context = KakarotTestEnvironmentContext::new(false).await;
```

Comment:
what is the meaning of the boolean? maybe we could name it above just for clarity?
```
let meaning_of_the_boolean = false;
let test_context = KakarotTestEnvironmentContext::new(meaning_of_the_boolean).await;
```

Reply:
the compiler would optimise anyway

Reply:
Gotcha will add a name 🫡